index.js

```diff
@@ -30,6 +30,7 @@ const config = {
 		'object-shorthand': 'error',
 		'prefer-const': 'error',
 		'quote-props': ['error', 'as-needed'],
+		'sort-keys': ['error', 'asc', {caseSensitive: false}],
```

Comment:
My only doubt is whether the Java sorting we do is case-insensitive too; if we do, then we should match that here, otherwise I'd say not. The thing I don't like about case-insensitive mode is that it isn't deterministic; ie. these are both valid:

```javascript
const a = {
  a: 1,
  A: 1,
}

const b = {
  A: 1,
  a: 1,
}
```

Reply:
Verified that Java sorting in portal seems to be "partially" insensitive in that this is the ordering I'm seeing:

```
fooa
fooA
fooa1
fooa11
fooa20
foob
fooB
```